docs/modules/ROOT/pages/vshn-managed/nextcloud/create.adoc

```diff
@@ -0,0 +1,57 @@
+= Create a Nextcloud service
+
+Apply the following object on your namespace, as specified by its YAML description.
+
+.Example to create a Nextcloud instance
+[source,yaml]
+----
+apiVersion: vshn.appcat.vshn.io/v1
+kind: VSHNNextcloud
+metadata:
+  name: nextcloud-app1-prod # <1>
+  namespace: prod-app # <2>
+spec:
+  parameters:
+    service:
+      version: "23" # <3>
+      defaultInternalDB: true # <4>
```

Comment:
we need to rethink this part that's very important!

Reply:
maybe we should use `managedPostgreSQL: true/false` and then `pgSettings`? Since we are pushing our PostgreSQL managed instance by default, there is not need to show the default DB. We can add that to the description of `managedPostgreSQL` parameter

Reply:
we agreed to use this name `useExternalPostgreSQL` variable and also reuse the [postgreSQLParameters](https://docs.appcat.ch/vshn-managed/keycloak/create.html)